### PR TITLE
Addition of comments

### PR DESCRIPTION
**Pull request by joinimran**
As requested by user comments has been updated to improve the structure of HTML of the document.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4918

### CODE REVIEW

**Comment by joinimran** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by illfated** [comment]
Rich diff: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/6013/files?short_path=1d7137d#diff-1d7137dbf2fc5383b8d875663c617788

Please note that the comment lines directly before any MarkDown code lines, like the 8 [!TIP] notes, makes it look like the MarkDown code has become broken or at least deactivated.

File: windows/client-management/mdm/bitlocker-csp.md [hunk 1]
```diff
@@ -1021,11 +1105,16 @@ Status code can be one of the following:
     <td><img src="images/crossmark.png" alt="cross mark" /></td>
 </tr>
 </table> 
+<!--/SupportedSKUs-->
 Value type is int. Supported operation is Get.
-
+<!--/Policy-->
+<!--Policy-->
 <a href="" id="status-rotaterecoverypasswordsrequestid"></a>**Status/RotateRecoveryPasswordsRequestID**
```
Comment (illfated) [inline windows/client-management/mdm/bitlocker-csp.md]: The preceding HTML tag line interferes with the MarkDown formatting code, breaking the **bold text** formatting.

File: windows/client-management/mdm/bitlocker-csp.md [hunk 2]
```diff
@@ -985,22 +1065,26 @@ This node reports compliance state of device encryption on the system.
...
 - 0 - Indicates that the device is compliant.
 - Any other value represents a non-compliant device.
-
+<!--/SupportedValues-->
 Value type is int. Supported operation is Get.
-
+<!--/Policy-->
+<!--Policy-->
 <a href="" id="status-rotaterecoverypasswordsstatus"></a>**Status/RotateRecoveryPasswordsStatus**  
+<!--Description-->
```
Comment (illfated) [inline windows/client-management/mdm/bitlocker-csp.md]: This HTML tag interferes with the MarkDown formatting code, breaking the NewLine formatting by concatenating the lines.

File: windows/client-management/mdm/bitlocker-csp.md [hunk 3]
```diff
@@ -985,22 +1065,26 @@ This node reports compliance state of device encryption on the system.
...
 This node reports the status of RotateRecoveryPasswords request. 
-
+<!--/Description-->
```
Comment (illfated) [inline windows/client-management/mdm/bitlocker-csp.md]: This HTML tag interferes with the MarkDown formatting code, breaking the NewLine formatting by concatenating the lines.

**Comment by joinimran** [comment]
Thank @illfated for pointing out these. Looks good to me now but give your opinion.

File: windows/client-management/mdm/bitlocker-csp.md [hunk 4]
```diff
@@ -1021,11 +1112,17 @@ Status code can be one of the following:
     <td><img src="images/crossmark.png" alt="cross mark" /></td>
 </tr>
 </table> 
+<!--/SupportedSKUs-->
 Value type is int. Supported operation is Get.
+<!--/Policy-->
```
Comment (illfated) [inline windows/client-management/mdm/bitlocker-csp.md]: This tag removes the spacing between this text line and the **bold text heading** below.

File: windows/client-management/mdm/bitlocker-csp.md [hunk 5]
```diff
@@ -985,22 +1069,29 @@ This node reports compliance state of device encryption on the system.
...
 </tr>
 </table> 
-
+<!--/SupportedSKUs-->
+<!--SupportedValues-->
 Supported values:  
 - 0 - Indicates that the device is compliant.
 - Any other value represents a non-compliant device.
-
+<!--/SupportedValues-->
```
Comment (illfated) [inline windows/client-management/mdm/bitlocker-csp.md]: This tag concatenates the list above, changing the bullet point list into a 1 line sentence.

File: windows/client-management/mdm/bitlocker-csp.md [hunk 6]
```diff
@@ -985,22 +1069,29 @@ This node reports compliance state of device encryption on the system.
...
 Value type is int. Supported operation is Get.
+<!--/Policy-->
```
Comment (illfated) [inline windows/client-management/mdm/bitlocker-csp.md]: This tag removes the spacing between this text line and the **bold text heading** below.

File: windows/client-management/mdm/bitlocker-csp.md [hunk 7]
```diff
@@ -957,14 +1037,18 @@ Each server-side recovery key rotation is represented by a request ID. The serve
...
 </tr>
 </table> 
+<!--/SupportedSKUs-->
 Value type is string. Supported operation is Execute. Request ID is expected as a parameter.
 
 <a href="" id="status"></a>**Status**  
 Interior node. Supported operation is Get.
-
-<a href="" id="status-deviceencryptionstatus"></a>**Status/DeviceEncryptionStatus**  
+<!--/Policy-->
```
Comment (illfated) [inline windows/client-management/mdm/bitlocker-csp.md]: This tag _and the tag before_ remove the spacing before and after the sub-heading **Status** .

File: windows/client-management/mdm/bitlocker-csp.md [hunk 8]
```diff
@@ -916,15 +987,23 @@ This setting initiates a client-driven recovery password refresh after an OS dri
     <td><img src="images/crossmark.png" alt="cross mark" /></td>
 </tr>
 </table> 
+<!--/SupportedSKUs-->
 Value type is int. Supported operations are Add, Delete, Get, and Replace.
-
+<!--SupportedValues-->
```
Comment (illfated) [inline windows/client-management/mdm/bitlocker-csp.md]: This tag removes the MD bullet point list functionality.

**Comment by illfated** [comment]
There are likely more tag issues breaking MarkDown in this document, but I need a couple of hours' break before pointing out more of them.

**Comment by joinimran** [comment]
@illfated Thanks for your help. I fixed those suggested by you and made some more fixes. Give it a quick look to see how does it looks like.

File: windows/client-management/mdm/bitlocker-csp.md (hunk 8 above)
Comment (illfated) [reply windows/client-management/mdm/bitlocker-csp.md]: This comment can be marked as **Resolved** .

File: windows/client-management/mdm/bitlocker-csp.md (hunk 7 above)
Comment (illfated) [reply windows/client-management/mdm/bitlocker-csp.md]: You may mark this comment as **Resolved** too.

File: windows/client-management/mdm/bitlocker-csp.md (hunk 5 above)
Comment (illfated) [reply windows/client-management/mdm/bitlocker-csp.md]: **Resolved**

File: windows/client-management/mdm/bitlocker-csp.md (hunk 6 above)
Comment (illfated) [reply windows/client-management/mdm/bitlocker-csp.md]: **Resolved**

File: windows/client-management/mdm/bitlocker-csp.md (hunk 4 above)
Comment (illfated) [reply windows/client-management/mdm/bitlocker-csp.md]: **Resolved**

**Comment by illfated** [comment]
Very good, well done. The resulting file can be viewed in Rich Text (with images) on this page:
- https://github.com/MicrosoftDocs/windows-itpro-docs/blob/16ea53ec106378ef094b470f63ca820c8593e528/windows/client-management/mdm/bitlocker-csp.md

**Comment by joinimran** [comment]
@Dansimp This PR is ready to go live.

**Pull request merged**